code/addons/docs/src/preset.ts

```diff
@@ -62,7 +66,7 @@ export async function webpack(
     babelOptions,
     mdxBabelOptions,
     configureJSX = true,
-    sourceLoaderOptions = { injectStoryParameters: true },
```

Comment:
Would it make sense to still accept this property with the purpose of warning users/throwing an error and directing them to the migration docs?

Reply:
Great idea! 💯